roles/vdm/library/siteconfig_info.py

```diff
@@ -2,38 +2,28 @@
 import glob
 import yaml
 import os
+from enum import Enum, auto
+
+class Overlay(Enum):
+  COMPONENT = "components"
+  CONFIGURATION = "configurations"
+  GENERATOR = "generators"
+  RESOURCE = "resources"
+  TRANSFORMER = "transformers"
```

Comment:
Just curious why are the defined items singular when the assigned values are plural?